.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,92 @@
+---
+description: Report a bug.
+labels: [bug]
+name: Bug
+title: "<SCOPE>: <SUMMARY>"
+
+body:
+  - type: "checkboxes"
```

Are these double quotes necessary?

```suggestion
  - type: checkboxes
```